src/sqlfluff/core/templaters/jinja.py

```diff
@@ -266,6 +270,21 @@ def get_context(self, fname=None, config=None, **kw) -> Dict:
                     if name not in live_context:
                         live_context[name] = dbt_builtins[name]
 
+            apply_airflow_builtins = config.get_section(
+                (self.templater_selector, self.name, "apply_airflow_builtins")
+            )
+            if apply_airflow_builtins:
+                import sqlfluff.core.templaters.jinja_builtins.airflow as airflow_builtins  # noqa
+
+                live_context.update(
```

Comment:
You asked about how to provide your module with access to `FluffConfig`. Could you add a line here, something like:
```
live_context["fluff_config"] = config
```

(In terms of design quality, it would be better to pass only the required portion of the config, rather than passing everything, so the above is just a starting point.)